benchmarks/buiter_et_al_2016_jsg/exp_2_high_resolution.prm

```diff
@@ -0,0 +1,210 @@
+#### This is the unstable wedge experiment 2
+###  Experiment 2 tests how an unstable subcritical wedge deforms to reach the critical taper
+###  solution. Horizontal layers of sand have 10 cm shortening by inward movement of a mobile
+###  wall with a velocity of 2.5 cm/hour. Both the base and the surface are horizontal and the 
```

Comment:
We actually do not have a true mobile wall, but rather approximate it through a rigid indenter.

Reply:
> We actually do not have a true mobile wall, but rather approximate it through a rigid indenter.

Got it. Thanks